RecoEcal/EgammaClusterAlgos/interface/SCEnergyCorrectorSemiParm.h

```diff
@@ -146,6 +147,7 @@ template <edm::Transition esTransition>
 void SCEnergyCorrectorSemiParm::setTokens(const edm::ParameterSet& iConfig, edm::ConsumesCollector cc) {
   isHLT_ = iConfig.getParameter<bool>("isHLT");
   isPhaseII_ = iConfig.getParameter<bool>("isPhaseII");
+  regTrainedWithPS_ = iConfig.getParameter<bool>("regTrainedWithPS");
```

Comment:
This parameter should be added to the `fillPSetDescription` as well.
Why is it set in this function and not in the constructor itself? It is not needed to set any token.